srcpkgs/kpkpass/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'kpkpass'
+pkgname=kpkpass
+version=21.04.0
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules qt5-qmake qt5-host-tools"
+makedepends="karchive-devel qt5-devel"
+short_desc="Apple Wallet Pass reader"
+maintainer="Louis Dupré Bertoni <contact@louisdb.xyz>"
+license="LGPL-2.1-or-later"
+homepage="https://kontact.kde.org"
+distfiles="${KDE_SITE}/release-service/${version}/src/${pkgname}-${version}.tar.xz"
+checksum=b117e2b04f66e101662ef85a56e2636d78b4092e897f14ff0c27ce61874c2e68
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" shared-mime-info"
```

Comment:
keep it unconditional. It is always needed, on native builds it is just pulled in by some makedepends.